src/krate.rs

```diff
@@ -598,6 +602,32 @@ impl Crate {
         Ok(users.chain(teams).collect())
     }
 
+    pub fn owner_team(&self, conn: &PgConnection) -> CargoResult<Vec<Owner>> {
```

Comment:
`Crate` has a lot of behavior on it as is. What do you think about putting this on `Team` instead and calling it `Team::owning(crate: &Crate, conn: &PgConnection)`